Sources/PerformanceTesting/PerformanceTestCase.swift

```diff
@@ -26,7 +26,8 @@ open class PerformanceTestCase: XCTestCase {
 
     // MARK: Instance Methods
 
-    /// Benchmarks the performance of a closure.
+    /// - Returns: An array of two-tuples containing the size of the structure and the average time
```

Comment:
Yeah, I agree. I feel like there could be a middle ground between `testPoint` and `size`.